tests/by-util/test_env.rs

```diff
@@ -150,3 +157,60 @@ fn test_fail_null_with_program() {
     let out = new_ucmd!().arg("--null").arg("cd").fails().stderr;
     assert!(out.contains("cannot specify --null (-0) with command"));
 }
+
+#[cfg(not(windows))]
```

Comment:
Could you please fix the fmt issue? thanks

Reply:
manually ran `rustfmt` on this file. Should look good now.